### ISSUE TITLE
Any way to mass upload / export

### ISSUE THREAD

**Issue by gouaille** [Happyr/TranslationBundle#28]
Hi Tobias and thank's for your amazing work.

I have a question about your bundle : is there any way to mass export the translations to loco ? Like when we're using the JMS Translator exporter. 

I don't know if i'm missing something or if you thought this another way :smile:

**Comment by Nyholm** [comment]
Hey. 
Thank you. 

I used Loco's import function to initially import all my existing translation. 

At some point I had translations in csv files so I used a script like [this](https://gist.github.com/Nyholm/dd6617e4cde9071a9eb2).

**Comment by gouaille** [comment]
Hey, thank's for the quick answer. 

So correct me if i'm wrong but the bundle is actually one-way synced right ? The Synchronize command is not able to get all the newly created string in the Symfony, you still have to add them one by one. 

Is this a feature you are interested by ?

**Comment by Nyholm** [comment]
It is a two way sync once you have everything set up. Hm.. or it depends how you look at it. I understand your point of view now.

When you run the sync command you are pulling from Loco. So it is a one way sync. But you can also add/edit/remove translation locally and send them (pretty much) one by one to Loco. 

I believe what we are looking for is an init command, is that correct? You want to push everything you have locally to Loco and start using their service, right?

**Comment by gouaille** [comment]
the init command could be useful but that's not what i'm really meaning. 

More like an "push" command which could be handle to send the modifications to loco without reimporting the files manually each time. 

My workflow actually is doing a translation:extract and then a happyr:translation:sync. Could be great to have something like a happyr:translation:push.

Don't know if my explanation is clear enough :smile:

**Comment by Nyholm** [comment]
I see. Okey. 
It is dangerous to to a push because it will overwrite everything on Loco. Say if a translator or an other developer made some changes in Loco without you knowing it...

I've also created an issue a while back on [JMSTranslationBundle](https://github.com/schmittjoh/JMSTranslationBundle/issues/283) in order to make sure we can reuse their great extract functionality. 

I think it would be a great improvement to create a push command. Would you mind sending a PR?

**Comment by gouaille** [comment]
Yeah that's exactly the point if i was afraid of : i do not want to overwrite loco, i'd like to do a sync. 
It seems loco is using special naming convention which could be really tricky to implement in symfony / your bundle.

I'll start with the import feature which will be useful anyway :smiley:

**Comment by gouaille** [comment]
And to answer your point about JMSTranslationBundle, it wasn't working properly with my custom exporter since JMS export to xliff and you are expecting .xlf and you are not supporting this extension.
I updated my setup to export .xlf files and it's working.
